src/org/zaproxy/zap/extension/accessControl/ZapAddOn.xml

```diff
@@ -1,6 +1,6 @@
 <zapaddon>
 	<name>Access Control Testing</name>
-	<version>6</version>
+	<version>7</version>
```

Comment:
This change should be reverted, instead update `changes` element mentioning that API support was added.